Tests/Tests/NSLocale+STPSwizzling.m

```diff
@@ -0,0 +1,59 @@
+//
+//  NSLocale+STPSwizzling.m
+//  StripeiOS Tests
+//
+//  Created by Cameron Sabol on 7/17/18.
+//  Copyright © 2018 Stripe, Inc. All rights reserved.
+//
+
+#import "NSLocale+STPSwizzling.h"
+
+#import <objc/runtime.h>
+
+@interface NSObject (STPSwizzling)
+
++ (void)stp_swizzleClassMethod:(SEL)original withReplacement:(SEL)replacement;
+
+@end
+
+@implementation NSObject (STPSwizzling)
+
++ (void)stp_swizzleClassMethod:(SEL)original withReplacement:(SEL)replacement
```

Comment:
Did you consider using the `STPAspects` code instead?

This is a really simple usage, and since it's test code I don't think it needs to get fancy, mostly just curious.

Reply:
didn't even know that existed! I'll keep in mind for next time